psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
```

Comment:
I probably didn't clean out my build directory correctly but removing the 'RUN_CCTRANSORT" option does cause this to throw.  Probably the source of Travis failures